docs/digger/admin/installing-jenkins-plugins.asciidoc

```diff
@@ -0,0 +1,21 @@
+== Installing Jenkins plugins
+Digger will by default install a number of Jenkins plugins required for its proper operation. If more Jenkins plugins are required, they can be specified in Ansible playbook(???) or using the Jenkins user interface.
```

Comment:
@aidenkeating I believe if we provide a `jenkins_plugins` value then it will override the existing one in the [defaults file](https://github.com/aerogear/digger-installer/blob/14aab018269a5e0613ceb07f26ba8fa4a46b8513/configure-buildfarm/defaults/main.yml#L13) We don't normally point the user to the defaults folder but in this case we may need to make an exception.